.github/workflows/merge.yml

```diff
@@ -7,14 +7,14 @@ name: Merge
 
 on:
   push:
-    branches: [ master, main ]
+    branches: [master, main]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container: [ "python:3.8", "python:3.9", "python:3.10" ]
+        container: ["python:3.8", "python:3.9", "python:3.10", "python:3.11"]
```

Comment:
Would it be sufficient to only test a minimum version like `3.8` and a maximum version like `3.11`?

Reply:
Good question, if we ever have any code that tests for version (using `sys.version_info` or similar) it does matter where we put the boundary, a mistake here could cause some branch of the code to run in an unsupported version.

This could also happen if we try-catch a package import but forget that a certain class/function is only available in a later version.

Reply:
We can leave this as is for now. If the tests take too long, we can explore reducing the list.